src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -24,45 +28,53 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
     requestInfo: RequestInfo,
     context: Context,
     services: Services
-  ): FutureHandlerResult =
-    services.zuoraService.getRecurringSubscription(state.user.id, state.product.billingPeriod).flatMap {
-      case Some(sub) => skipSubscribe(state, requestInfo, sub)
+  ): FutureHandlerResult = for {
+    identityId <- Future.fromTry(IdentityId(state.user.id))
```

Comment:
it would be better if all the converting from the Wire model (Strings etc) to the domain model (IdentityId etc) would happen up front to keep things simple at this level.  But I wanted to just do the simplest change so I am just flatmapping in the one field.